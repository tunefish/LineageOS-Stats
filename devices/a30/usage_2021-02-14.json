{
  "17.1": 29,
  "18.0": 640,
  "18.1": 196,
  "none": 4
}