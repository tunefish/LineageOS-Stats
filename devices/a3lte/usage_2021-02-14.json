{
  "15.1": 583,
  "16.0": 197,
  "17.1": 651
}