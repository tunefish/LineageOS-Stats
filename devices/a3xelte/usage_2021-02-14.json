{
  "15.1": 112,
  "16.0": 461,
  "17.1": 3464,
  "18.0": 3,
  "18.1": 13,
  "52.8": 1,
  "none": 2
}