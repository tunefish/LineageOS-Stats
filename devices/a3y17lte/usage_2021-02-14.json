{
  "14.1": 5453,
  "15.1": 341,
  "16.0": 1002,
  "17.1": 953,
  "18.0": 1
}