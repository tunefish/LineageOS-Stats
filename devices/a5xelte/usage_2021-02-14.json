{
  "15.1": 55,
  "16.0": 524,
  "17.1": 4170,
  "18.0": 12,
  "18.1": 2,
  "none": 1
}