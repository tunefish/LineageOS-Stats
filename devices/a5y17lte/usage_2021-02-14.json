{
  "14.1": 225,
  "15.1": 130,
  "16.0": 762,
  "17.1": 4683,
  "none": 1
}