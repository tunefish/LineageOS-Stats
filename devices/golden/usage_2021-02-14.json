{
  "13.0": 1022,
  "14.1": 3352,
  "15.1": 4,
  "17.1": 5,
  "none": 1
}